### PR TITLE
Add European Structural and Investment funds to Publisher

### PR DESCRIPTION
**Pull request by jackscotti**
This PR contains the work necessary to enable creation of DCLG/European Structural and Investment funds from Specialist Publisher. 
The facets will be added in a different PR.

Ticket: https://www.pivotaltracker.com/story/show/87188568
Totally inspired by this great PR: https://github.com/alphagov/specialist-publisher/pull/400

### CODE REVIEW

File: app/exporters/formatters/esi_fund_indexable_formatter.rb [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+
+class EsiFundIndexableFormatter < AbstractSpecialistDocumentIndexableFormatter
+
+  def type
+    "european_structural_investment_fund"
+  end
+
+private
+  def organisation_slugs
+    []
```
Comment (tommyp) [inline app/exporters/formatters/esi_fund_indexable_formatter.rb]: No Organisation?

File: features/support/env.rb [hunk 2]
```diff
@@ -97,40 +97,44 @@
...
 
+require "aaib_report_helpers"
+require "cma_case_helpers"
+require "csg_helpers"
+require "dsu_helpers"
+require "esi_fund_helpers"
+require "idf_helpers"
+require "maib_report_helpers"
+require "msa_helpers"
+require "raib_report_helpers"
```
Comment (tommyp) [inline features/support/env.rb]: :+1: to alphabetising and moving these to have whitespace

File: app/exporters/formatters/esi_fund_indexable_formatter.rb (hunk 1 above)
Comment (jackscotti) [inline app/exporters/formatters/esi_fund_indexable_formatter.rb]: We have been told that there won't be one

Comment (tommyp) [inline app/exporters/formatters/esi_fund_indexable_formatter.rb]: Ah right. No worries. Just wanted to check.

**Comment by tommyp** [comment]
LGTM

**Comment by jackscotti** [comment]
Thanks!

**Pull request merged**